scriptTest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+if [ "$1" = "install" ]; then
+	clear
+	echo "=========== Begin to install and build ========="
+	sudo python3.4 setup.py install
+	sudo python3.4 setup.py build
+	echo "=========== Installed and built the sniffles ========="
+	echo
+elif [ "$1" = "test" ]; then
+	clear
+	echo "=========== Begin to install and build ========="
+	sudo python3.4 setup.py install
```

Comment:
I misunderstood your intention. I thought the purpose of `test` is to run unit tests. If you want to test installed software,
I suggest that your script should run `setup.py build` and `runtests.py` first, and then, only if those unit tests are successful, run `setup.py install` to install those files into the system.